packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -82,7 +82,7 @@ export class ModelAuthTransformer extends Transformer {
     constructor(config?: ModelAuthTransformerConfig) {
         super(
             'ModelAuthTransformer',
-            `
+            gql`
```

Comment:
When customers have graphql syntax tools installed in editors/IDEs like VSCode, using this template function will cause the editor to syntax highlight the contents. This makes for a better UX when writing custom transformers.